tools/screenshot-test/functions/data_image.ts

```diff
@@ -0,0 +1,39 @@
+import * as firebaseFunctions from 'firebase-functions';
+import {writeFileSync} from 'fs';
+import {verifySecureToken} from './jwt_util';
+import {isCreateEvent} from './util/util';
+
+const gcs = require('@google-cloud/storage')();
+
+/** The storage bucket to store the images. The bucket is also used by Firebase Storage. */
+const bucket = gcs.bucket(firebaseFunctions.config().firebase.storageBucket);
+
+/**
+ * Convert data to images. Image data posted to database will be saved as png files
```

Comment:
Is the incoming data a base-64 encoded image? If so, the comment should say something like 

"Writes base-64 encoded test images to png files on the filesystem."

I'd also rename the function to `writeTestImagesToFiles`